workloads/kube-burner/workloads/pod-density-heavy/templates/pod.yml

```
@@ -0,0 +1,33 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: {{.JobName}}-{{.Iteration}}
+  labels:
+    name: {{.JobName}}
+spec:
+  nodeSelector: {{.nodeSelector}}
+  containers:
+  - name: pod-density-heavy
+    image: {{.containerImage}}
+    ports:
+    - containerPort: 8080
+      protocol: TCP
+    imagePullPolicy: IfNotPresent
+    readinessProbe:
+      httpGet:
+        path: /
+        port: 8080
+      periodSeconds: {{ .readinessPeriod }}
+      failureThreshold: 1
+      timeoutSeconds: 60
+      initialDelaySeconds: 30
+    livenessProbe:
```

I wonder if we can add an extra "exec" livenessprobe https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command to make the workload _heavier_

```suggestion
    livenessProbe:
      httpGet:
        path: /
        port: 8080
      exec:
        command:
        - echo
```